Makefile

```diff
@@ -32,6 +32,7 @@ build:
 
 .PHONY: generate
 generate:
+	go install ./vendor/golang.org/x/tools/cmd/stringer
```

Comment:
![](http://www.reactiongifs.com/r/but-why.gif)

is this just something we missed before?